DOC-3321 Added upgrade notes and skipSizeQuery to config table (#299)
corda/corda-docs-portal: content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md

* Business Network Operators (BNOs)
* Corda developers

> ## Introducing Collaborative Recovery V1.2.1

Reviewer: ```suggestion

## Introducing Collaborative Recovery V1.2.1
```

Before installing and using the Collaborative Recovery CorDapps, you need to know:

* Your Business Network disaster recovery policy.
* The Corda platform requirements.
* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used.

## In this version

You can now use the `skipSizeQuery` configuration in `LedgerRecover` in cases where you cannot grant `SELECT` permission by the PostgreSQL admin to check the file size of the exported transactions before exporting them. This check normally verifies that the size of files to be exported does not exceed your node’s disk size.

This change does not remove the file size limit from a network, so if the exported transactions do exceed your node's disk size, the recovery operation can still fail. In this case, you will get a generic error from Corda, rather than a specific reason for the recovery failure.

### Changes in V1.2

> In Collaborative Recovery V1.2, LedgerSync has been modified to be compatible with the [Archive Service](/en/platform/corda/4.8/enterprise/node/archiving/archiving-setup.md).

Reviewer: ```suggestion
In Collaborative Recovery V1.2, LedgerSync has been modified to be compatible with the [Archive Service](../../../../../../../en/platform/corda/4.8/enterprise/node/archiving/archiving-setup.md).
```

Before installing and using the Collaborative Recovery CorDapps, you need to know:

* Your Business Network disaster recovery policy.
* The Corda platform requirements.
* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used.

## In this version

> You can now use the `skipSizeQuery` configuration in the Ledger Recover CorDapp in cases where you cannot grant `SELECT` permission to PostGres admin to check the file size of the exported transactions before exporting them. This check normally verifies that the size of files to be exported does not exceed your node’s disk size.

Reviewer 2: ```suggestion
You can now use the `skipSizeQuery` configuration in the Ledger Recover CorDapp in cases where you cannot grant `SELECT` permission to PostGres admin to check the size of artifacts (transactions, attachments, network parameters). In Automatic Ledger Recover, this check normally verifies that the size of transactions (including their backchain with attachments and network parameters) does not exceed the maximum allowed message size. In Manual Ledger Recover, this check normally verifies that the size of files to be exported does not exceed your node’s available disk size. In Manual Ledger Recover, 
```

> You can now use the `skipSizeQuery` configuration in the Ledger Recover CorDapp in cases where you cannot grant `SELECT` permission by the PostGres admin to check the file size of the exported transactions before exporting them. This check normally verifies that the size of files to be exported does not exceed your node’s disk size.

Reviewer: A minor thing but should "Ledger Recover CorDapp" be `LedgerRecover` (as per https://docs.r3.com/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.html#ledgerrecover)?

Reviewer: Also: PostGres > PostgreSQL

This change does not remove the file size limit from a network, so if the exported transactions do exceed your node's disk size, the recovery operation can still fail. In this case, you will get a generic error from Corda, rather than a specific reason for the recovery failure. 

### Changes from V1.2

> In Collaborative Recovery V1.2, LedgerSync has been modified to be compatible with the [Archive Service](../archiving/archiving-setup).

Reviewer: Does this link work?
